modules/darwin/common/default.nix

```diff
@@ -36,6 +36,9 @@
     htop
   ];
 
+  # disable application layer firewall, telegraf needs an incoming connection
+  system.defaults.alf.globalstate = 0;
```

Comment:
This was already disabled on the scaleway machine when I was setting it up but it was enabled on the hetzner machines.